### DISCUSSION TITLE
Eclipse Dirigible 8.0 will be on Spring Boot only

### DISCUSSION THREAD

**Discussion by delchev** [eclipse/dirigible#2390] [Announcements]
Hi All,

As you may know since a few months we are moving main backend functionality to a Spring Boot stack under the `components` folder. Testing and feedback is very much welcome at this point.

More about it you can find at: https://github.com/eclipse/dirigible/tree/master/components#readme

The issue containing the links to commits is: #2107 

The release 8.0.0 will contain only this distribution and all the others will be discontinued.

> Please, prepare accordingly your target environments.

Regards,
Nedelcho

**Comment by ThuF** [comment]
As part of the migration to Spring Boot, the BPMN process definitions _(***.bpmn** files)_ should be updated as follows:

```xml
<serviceTask id="processId" name="TaskName" flowable:class="org.eclipse.dirigible.bpm.flowable.DirigibleCallDelegate">
```
->

```xml
<serviceTask id="processId" name="TaskName" flowable:class="org.eclipse.dirigible.components.engine.bpm.flowable.delegate.DirigibleCallDelegate">
```

<img width="882" alt="Screenshot 2023-05-23 at 18 02 47" src="https://github.com/eclipse/dirigible/assets/4092083/ef17143a-a19e-4618-ba17-d72ffc608a9a">
